CONTRIBUTING.md

```diff
@@ -33,6 +33,11 @@ before implementing major changes to torchjd.
      pdm run pytest tests/unit
      ```
 
+   - If you have access to a cuda-enabled GPU, you should also check that the unit tests pass on it:
+     ```bash
+     CUBLAS_WORKSPACE_CONFIG=:4096:8 PYTEST_TORCH_DEVICE=cuda pdm run pytest tests/unit
```

Comment:
The `4096:8` isn't specific to your GPU? I don't know how this works.

Reply:
Not really. It might not be available in older versions of cuda, but apart from that it's pretty standard from what I see online.

Reply:
See: https://docs.nvidia.com/cuda/cublas/index.html#results-reproducibility

There are two possibilities: `:16:8` or `:4096:8`. I think it's related to how much memory some internal cuda algorithm is allowed to use.